upstream_utils/update_llvm.py

```diff
@@ -126,11 +126,10 @@ def overwrite_files(wpiutil_files, llvm_files):
 
     for wpi_file in wpiutil_files:
         wpi_base_name = os.path.basename(wpi_file)
-        if wpi_base_name not in llvm_files:
-            if wpi_base_name not in unmatched_files_whitelist:
-                print(f"No file match for {wpi_file}, check if LLVM deleted it")
-        else:
+        if wpi_base_name in llvm_files:
```

Comment:
```suggestion
```suggestion
        if wpi_base_name not in llvm_files and wpi_base_name not in unmatched_files_whitelist:
              print(f"No file match for {wpi_file}, check if LLVM deleted it")
              continue

        shutil.copyfile(llvm_files[wpi_base_name], wpi_file)
```
```